pkgs/applications/misc/buku/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, python3, pythonPackages, fetchFromGitHub,
+encryptionSupport ? false
+}:
+
+pythonPackages.buildPythonApplication rec {
+    version = "1.8";
+    name = "buku-${version}";
+
+    src = fetchFromGitHub {
+        owner = "jarun";
+        repo = "buku";
+        rev = "53d48ee56a3abfb53b94ed25fb620ee759141c96";
+        sha256 = "185d3gndw20c3l6f3mf0iq4qapm8g30bl0hn0wsqpp36vl0bpq28";
+    };
+
+    propagatedBuildInputs = [ python3 ];
+
+    buildInputs = (if encryptionSupport
+            then [ pythonPackages.pycrypto ]
+            else [ ]);
+
+    phases = "unpackPhase installPhase";
```

Comment:
This should be a list of strings: `[ "unpackPhase" "installPhase" "fixupPhase" ]`

`buildPythonApplication` has a `fixupPhase` which patches all shebangs. This means it will replace `#!/... python' with the store path of the selected python binary. This makes the app run without having python installed in your current profile.